.github/workflows/test.yml

```
@@ -0,0 +1,103 @@
+name: Tests
```

```suggestion
name: Build/Lint/Test
```

Naming is hard, `Tests` doesn't really describe what's happening in this workflow though.